src/styles/stars.scss

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 4px;
+  }
+
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
```

use the power of nesting
```suggestion
  
  &--1 .stars__star:nth-child(-n + 1),
  //...
}
```